transport/src/main/resources/stream-protocol.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+  ~ Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+  ~ one or more contributor license agreements. See the NOTICE file distributed
+  ~ with this work for additional information regarding copyright ownership.
+  ~ Licensed under the Zeebe Community License 1.1. You may not use this file
+  ~ except in compliance with the Zeebe Community License 1.1.
+  -->
+<sbe:messageSchema xmlns:sbe="http://fixprotocol.io/2016/sbe"
+  xmlns:xi="http://www.w3.org/2001/XInclude" package="io.camunda.zeebe.transport.stream.impl.messages"
+  id="0" version="1" semanticVersion="${project.version}"
+  description="Zeebe Protocol" byteOrder="littleEndian">
+
+  <xi:include href="../../../protocol/src/main/resources/common-types.xml"/>
```

Comment:
:thought_balloon: I wish we didn't have to keep generating them over and over and over :smile: